bot/cogs/fun.py

```diff
@@ -199,6 +199,43 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @commands.command(aliases=["leet"])
+    async def Leetify(self, ctx):
+        Leetters = {'a':['A','4'], 'b':['b', 'B'], 'c':['C', 'c'], 'd':['D', 'd'],
+                        'e':['E', '3'], 'f':['F', 'f'], 'g':['G', 'g'], 'h':['H', 'h'],
+                        'i':['1', 'I', 'i'], 'j':['j', 'J'], 'k':['K', 'k'], 'l':['L', 'l'],
+                        'm':['M', 'm'], 'n':['n', 'N'], 'o':['O', '0'], 'p':['P', 'p'],
+                        'q':['Q', 'q'], 'r':['R', 'r'], 's':['5', 'S'], 't':['t', 'T'],
+                        'u':['U', 'u'], 'v':['v','V'], 'w':['W', 'w'], 'x':['X', 'x'],
+                        'y':['y', 'Y'], 'z':['Z', 'z']}
+        content = [i.lower()+' ' for i in [i for i in ctx.message.content.split(" ") if i.lower() != ">leet" and i.lower() != ">leetify"]]
+        strContent = ''
```

Comment:
It is very non-standard to split variables with capital letter separation for a new word, instead, the pythonic way to do it would be:
```suggestion
        str_content = ''
```
This also applies to all variables like this.

Reply:
As I've already said:
> **This also applies to all variables like this and they need to be renamed accordingly.**

So please, don't mark it as resolved until those are fixed too, also note that by directly committing this you've only changed the definition of `strContent` to `str_content` but not the references to that variables, which will result in an error.